lua/lspconfig/configs/clangd.lua

```diff
@@ -46,33 +46,29 @@ local function symbol_info()
   end, bufnr)
 end
 
-local root_files = {
-  '.clangd',
-  '.clang-tidy',
-  '.clang-format',
-  'compile_commands.json',
-  'compile_flags.txt',
-  'configure.ac', -- AutoTools
-}
-
-local default_capabilities = {
-  textDocument = {
-    completion = {
-      editsNearCursor = true,
-    },
-  },
-  offsetEncoding = { 'utf-8', 'utf-16' },
-}
-
 return {
   default_config = {
     cmd = { 'clangd' },
     filetypes = { 'c', 'cpp', 'objc', 'objcpp', 'cuda', 'proto' },
     root_dir = function(fname)
-      return util.root_pattern(unpack(root_files))(fname) or util.find_git_ancestor(fname)
+      return util.root_pattern(
+        '.clangd',
+        '.clang-tidy',
+        '.clang-format',
+        'compile_commands.json',
+        'compile_flags.txt',
+        'configure.ac' -- AutoTools
+      )(fname) or util.find_git_ancestor(fname)
     end,
     single_file_support = true,
-    capabilities = default_capabilities,
+    capabilities = {
+      textDocument = {
+        completion = {
+          editsNearCursor = true,
+        },
+      },
+      offsetEncoding = { 'utf-8', 'utf-16' },
```

Comment:
now the generated docs are more useful:

<img width="497" alt="image" src="https://github.com/user-attachments/assets/f92ab833-2aac-447f-afcc-8303f489f015">